documentation/source/conf.py

```diff
@@ -173,7 +173,7 @@
 
 # The name of an image file (relative to this directory) to place at the top
 # of the sidebar.
-# html_logo = None
+html_logo = '_static/cocotb-logo-white.svg'
```

Comment:
This will make Black happy.

```suggestion
html_logo = "_static/cocotb-logo-white.svg"
```